ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml

```diff
@@ -114,19 +114,13 @@ objects:
 
         trap 'rc=$?; CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi; if test "${rc}" -ne 0; then touch /tmp/shared/exit; fi; exit "${rc}"' EXIT
 
-        if [[ "${CLUSTER_TYPE:-}" != "aws" ]]; then
-          echo "[WARNING] No resource leasing is defined for \"${CLUSTER_TYPE}\" clusters"
-          touch /tmp/shared/leased
-          exit 0
-        fi
-
         # hack for bazel
         function boskosctl() {
           /app/boskos/cmd/cli/app.binary "${@}"
         }
 
         echo "[INFO] Acquiring a lease ..."
-        resource="$( boskosctl --server-url http://boskos.ci --owner-name "${CLUSTER_NAME}" acquire --type aws-quota-slice --state free --target-state leased --timeout 150m )"
+        resource="$( boskosctl --server-url http://boskos.ci --owner-name "${CLUSTER_NAME}" acquire --type "${CLUSTER_TYPE}-quota-slice" --state free --target-state leased --timeout 150m )"
```

Comment:
what does this look like if we add a new cluster type but forget to add a lease class for it?  Will we fail with a reasonable message, or do we need some sort of guard here to keep us from forgetting?

Reply:
[vsphere][1]:

```
failed to acquire a resource: resources not found
```

Good enough for an error message, but we'll want a high quota for this too.

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4699/rehearse-4699-pull-ci-openshift-installer-master-e2e-vsphere/1

Reply:
And [GCP][1]?  Somehow...  Ah, maybe because we aren't reconfiguring boskos for the rehearsal?

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4699/rehearse-4699-pull-ci-openshift-installer-master-e2e-gcp/1